scst-policy/configuring.md

```diff
@@ -1,205 +1,142 @@
 
-* `spec.verification.keys`: A list of public keys complementary to the private
-keys that were used to sign the images.
-
-* `spec.verification.images[].namePattern`: Image name patterns that the policy enforces.
-Each image name pattern maps to the required public keys. (Optional) Use a secret to authenticate the private registry where images and signatures matching a name pattern are stored.
-
-* `spec.verification.exclude.resources.namespaces`: A list of namespaces where
-this policy is not enforced.
-
-System namespaces specific to your cloud provider may need to be excluded from the policy. VMware also recommends configuring exclusions for Tanzu Application Platform system namespaces. This prevents the Image Policy Webhook from blocking components of Tanzu Application Platform.
-
-To get a list of created namespaces, run:
-
-  ```console
-  kubectl get namespaces
-  ```
-
-Tanzu Application Platform system namespaces can include:
-
-  ```console
-  - accelerator-system
-  - api-portal
-  - app-live-view
-  - app-live-view-connector
-  - app-live-view-conventions
-  - build-service
-  - cartographer-system
-  - cert-injection-webhook
-  - cert-manager
-  - conventions-system
-  - developer-conventions
-  - flux-system
-  - image-policy-system
-  - kapp-controller
-  - knative-eventing
-  - knative-serving
-  - knative-sources
-  - kpack
-  - learning-center-guided-ui
-  - learning-center-guided-w01
-  - learningcenter
-  - metadata-store
-  - scan-link-system
-  - secretgen-controller
-  - service-bindings
-  - services-toolkit
-  - source-system
-  - spring-boot-convention
-  - stacks-operator-system
-  - tanzu-cluster-essentials
-  - tanzu-package-repo-global
-  - tanzu-system-ingress
-  - tap-gui
-  - tap-install
-  - tap-telemetry
-  - tekton-pipelines
-  - triggermesh
-  ```
-
-The following is an example `ClusterImagePolicy`:
-
-  ```yaml
-  ---
-  apiVersion: signing.apps.tanzu.vmware.com/v1beta1
-  kind: ClusterImagePolicy
-  metadata:
-      name: image-policy
-  spec:
-    verification:
-      exclude:
-        resources:
-          namespaces:
-          - kube-system
-          - <TAP system namespaces>
-      keys:
-      - name: first-key
-        publicKey: |
-          -----BEGIN PUBLIC KEY-----
-          ...
-          -----END PUBLIC KEY-----
-      images:
-      - namePattern: registry.example.org/myproject/*
-        keys:
-        - name: first-key
-      - namePattern: registry.example.org/authproject/*
-        secretRef:
-          name: secret-name
-          namespace: namespace-name
-        keys:
-        - name: first-key
-  ```
+* `images`: The images block defines the patterns of images that should be
+  subject to the ClusterImagePolicy. If multiple policies match a particular
+  image, _ALL_ of those policies must be satisfied for the image to be admitted.
+  If there is no host in the `glob` field, `index.docker.io` is used for the
```

Comment:
Should it be noted that if there is no "repository" specified, it defaults to "library"?
i.e. if you use `busybox` it becomes `index.docker.io/library/busybox`?